oracc_rest/search.py

```diff
@@ -179,19 +181,25 @@ def complete(self, word):
         found in the data.
         """
         search = Search(using=self.client, index=self.index)
-        search = search.suggest("sug_complete",
-                                word,
-                                completion={
-                                    "field": "completions",
-                                    "skip_duplicates": True,
-                                    "size": 10}  # TODO how to get all?
-                                )
-        completion_results = search.execute().suggest.to_dict()['sug_complete']
+        search = search.suggest(
+            "sug_complete",
+            word,
+            completion={
+                "field": "completions",
+                "skip_duplicates": True,
+                "size": 10,
+            },  # TODO how to get all?
+        )
+        completion_results = search.execute().suggest.to_dict()["sug_complete"]
 
         all_completions = [
-            option["text"]
-            for option
-            in completion_results[0]["options"]
+            option["text"] for option in completion_results[0]["options"]
         ]
 
         return all_completions
+
+    def all_suggest_compiler(completions, suggestions):
+        """This combines the suggestions and completions into
+        a dictionary which can be displayed."""
+        results = {"completions": completions, "suggestions": suggestions}
+        return results
```

Comment:
I'm not sure this needs to be a method on the `ESearch` class - I think it would be more suitable to put it in the `__init__.py` file, as a plain function (like we currently have `_parse_request_args`. The reason is that it doesn't have to do with any communication with Elasticsearch, but rather with how we prepare the response to return.

(It's also simple enough that it could be part of the `get` method in `CombinedSuggestions`, but it's not a bad idea to have it as a separate function)

Reply:
I'll move it into the right file. The `'` to `"` is just from an auto-formatter but I could try change the settings if there's a preference?